etc/WMAgentConfig.py

```diff
@@ -55,7 +55,7 @@
              'cmssrm-fzk.gridka.de', 'ccsrm.in2p3.fr', 'srmcms.pic.es', 'cmssrmdisk.fnal.gov']
 
 # Job retry information.  This includes the number of times a job will be retried and
-# for how long it will sit in cool off.
+# for how long it will sit in cool off. maxJobRetries only used in T0 agent
 maxJobRetries = {'default': 3, 'Merge': 4, 'LogCollect': 2, 'Cleanup': 2, 'Harvesting': 2}
```

Comment:
Given that @hufnagel said T0 overwrites it, how about we completely delete it from this config then?